tests/unit/anchore_engine/subsys/events/test_factories.py

```diff
@@ -0,0 +1,70 @@
+import pytest
```

Comment:
I would have liked to add more unit tests to this PR, but being able to do so on the analyzer and catalog code that has been changed I think requires them to be refactored into more-atomic methods, which I do not consider in scope for this change/release.